operations/rates.md

```diff
@@ -104,6 +104,123 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `AvailabilityBlockAssignments` | array of [Availability block assignment](#availability-block-assignment) | optional | Shows which rates relate to which availability blocks. |
 | `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
 
+## Add rates
+
+Adds rates the enterprise. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/rates/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+    "Rates":[
+      {
+          "Names": { "EN": "My rate" },
+          "Type": "Public",
+          "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94"
+          "RateGroupId": "b9f25a45-9b9a-4b33-99bd-b06f008eb6f5",          
+          "AccountingCategoryId": "3620c660-a4ec-4e0f-a0bc-b06f008eb8bf",
+          "PricingType": "DependentRatePricing",
+          "Pricing": {
+              "DependentRatePricing": {
+                  "BaseRateId": "1a1ddd3b-e106-4a70-aef1-54a75b483943",
+                  "RelativeAdjustment": 0.15,
+                  "AbsoluteAdjustment": 10
+              }
+          }
+      }
+  ]
+}
+```
```

Comment:
There are several properties for which there are no JSON examples. That may be ok, but perhaps you could just review them and decide whether any need to be added.